src/main/java/ui/pages/TrelloHomePage.java

```diff
@@ -3,9 +3,11 @@
 import lombok.Getter;
 import ui.common.BasePage;
 import ui.fragments.AllBoardsFragment;
+import ui.fragments.AllClosedBoardsFragment;
 
+@Getter
```

Comment:
What happens if there is a private field that does not require a getter?

Reply:
I got it where you said it in your workplace, I'll change it